amimals/QA_27_7_3_Tuflina.java

```diff
@@ -0,0 +1,99 @@
+package amimals;
+
+import java.util.Random;
+import java.util.Scanner;
+
+/// Написать вроде написала, а вот как сделать
+/// читаемым и уменьшить не уверена что знаю
+
+public class QA_27_7_3_Tuflina {
+    public static void main(String[] args) {
+        Cats cats = new Cats(1, "Cat Boris", 9);
+        Crows crows = new Crows(1, "Black Raven", "Black");
+        Dogs dogs = new Dogs(1, "Dog Muhtar", 5);
+        Fish fish = new Fish(1, "Fish Dora", "Salmon");
+        Rabbits rabbits = new Rabbits(1, "Rabbit Snow", "white");
+        Tigers tigers = new Tigers(1, "Tiger Rik", "caspian");
+
+        Scanner scanner = new Scanner(System.in);
+        int size;
+        do {
+            System.out.print("Введите размер массива: ");
+            size = scanner.nextInt();
+            if (size <= 0)
+                System.out.println("Incorrect size of array! Try again!");
+        } while (size < 0);
+        String[] animalsArray = {"Cat ", "Raven ", "Dog ", "Fish ", "Rabbit ", "Tiger "};
+        String[] animals = new String[size];
+
+        Random random = new Random();
+        for (int i = 0; i < size; i++) {
+            int randomAnimal = random.nextInt(animalsArray.length);
+            animals[i] = animalsArray[randomAnimal];
+        }
+        for (int i = animals.length - 1; i > 0; i--) {
+            int index = random.nextInt(i + 1);
+            String temp = animals[i];
+            animals[i] = animals[index];
+            animals[index] = temp;
+        }
```

Comment:
A really interesting action. Yes, if array must have each animal type, it is a great solution, but it wasn't necessary)
Also seems random is not so ideal, but it is a math question